src/utils/hooks/useControl.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+import { useService } from '@xstate/react'
+import { controlService } from 'state/control'
+
+export const useControl = (control: 'arm' | 'flipper' | 'nothing') => {
+  const [, send] = useService(controlService)
+  useEffect(() => {
+    switch (control) {
+      case 'arm':
+        send('CONTROL_ARM')
+        break
+      case 'flipper':
+        send('CONTROL_FLIPPER')
+        break
+      case 'nothing':
```

Comment:
I find the naming a little confusing, Nothing isn't equal to DISABLED in my mind.

Reply:
I agree, the original idea was that you control nothing so I called the state nothing, but it's effectively the same as disabled in this context